lib/color_guard/store/redis.rb

```diff
@@ -16,6 +16,17 @@ def find(feature_name)
         end
       end
 
+      def all
+        features = {}
+        connection_pool.with do |redis|
+          keys = redis.keys
+          keys.each do |key|
+            features[key.to_sym] = Feature.new(key, redis.get(key))
+          end
```

Comment:
I think you should use `mget` so its less calls to redis and we can release the connection back to the poll faster